### ISSUE TITLE
Robot accelerometer and gyro use example

### ISSUE THREAD

**Issue by zayfod** [zayfod/pycozmo#17]
A new example is needed that demonstrates the use of Cozmo's:
- accelerometers for measuring acceleration along the x, y, and z axes
- gyro for measuring agular velocity around the x, y, and z axes

**Comment by ElectricTurtle32** [comment]
Yes this would be nice, also add an example for cube  acceleration

**Comment by donnie58744** [comment]
I can try to get this done
